pom.xml

```diff
@@ -15,7 +15,7 @@
 
 	<groupId>org.openmrs.module</groupId>
 	<artifactId>legacyui</artifactId>
-	<version>1.3.3</version>
```

Comment:
Since when did this change from https://github.com/openmrs/openmrs-module-legacyui/blob/master/pom.xml#L18